Client.py

```diff
@@ -118,6 +127,7 @@ def Handle(self, data):
 			msg_limits = self.floodlimit[self.access]
 		else:
 			msg_limits = self.floodlimit['user']
+		print("< [" + self.username +"] " + data)
```

Comment:
are you sure you want these prints? (one on line 227 as well)

Reply:
Oh, yes, those debug prints can go now.

Reply:
other than that, maybe squash this into a single commit? it's a lot of commits for the actual changes;
up to you and abma though